packages/components/src/components/Tooltip/TooltipContent.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+
+import { Tooltip, TooltipContent } from './Tooltip';
+
+type Story = StoryObj<typeof TooltipContent>;
+const meta: Meta<typeof TooltipContent> = {
+    component: TooltipContent,
+    title: 'Components/Tooltip/Content',
+    tags: ['autodocs'],
+    args: {
+        padding: 'spacious',
+        side: 'top',
+        ignoreCollisions: false,
+        maxWidth: '200px',
+        children:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    },
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+
+    render: ({ ...args }) => {
+        return (
+            <Tooltip.Root>
+                <Tooltip.Trigger>
+                    <Button>Hover over me!</Button>
+                </Tooltip.Trigger>
+                <Tooltip.Content {...args} />
+            </Tooltip.Root>
+        );
+    },
+};
+export default meta;
+
+export const Default: Story = {
```

Comment:
And we need to rename this to avoid conflicts of name (default exists in `Tooltip.stories.tsx`)
```suggestion
export const LongText: Story = {
```